src/docs/user/references/Variables.adoc

```diff
@@ -0,0 +1,487 @@
+
+Variables can come at hand while developing ProActive applications.
+Propagate data between tasks, customize jobs, and store useful debug information are a few examples of
+how variables can ease development. ProActive has basically two groups of variables :
+
+- <<_workflow_variables>>, declared on the XML job definition.
+- <<_dynamic_variables>>, system variables or variables created at task runtime.
+
+[[_workflow_variables]]
+==== Workflow variables
+
+Workflow variables are declared in the XML job definition. Within the definition we can have variables in several
+levels, at job level the variables are visible and shared by all tasks. At task level, variables are visible
```

Comment:
"we can have variables in several levels" --> maybe say at two levels (since you give only two options after)